aws/rust-runtime/aws-types/Cargo.toml

```diff
@@ -8,11 +8,10 @@ license = "Apache-2.0"
 repository = "https://github.com/awslabs/smithy-rs"
 
 [features]
-default = []
 hardcoded-credentials = []
 
 [dependencies]
-aws-smithy-async = { path = "../../../rust-runtime/aws-smithy-async" }
+aws-smithy-async = { path = "../../../rust-runtime/aws-smithy-async", features = ["rt-tokio"] }
```

Comment:
Why is `rt-tokio` required for `aws-types`?

Reply:
it probably isn't, I'll remove it

Reply:
ah, it does depend on the async sleep impl in `aws-smithy-async` so there will be some dep eventually but I think it's fine to remove this until we actually add other sleep impls.